packages/dnb-eufemia/src/components/input-masked/style/dnb-input-masked.scss

```
@@ -10,3 +10,83 @@
     font-family: var(--font-family-monospace);
   }
 }
+
+// MultiInputMask
+.dnb-multi-input-mask__fieldset {
+  margin: 0;
+  padding: 0;
+  border: none;
```

```suggestion
@include fieldsetReset();
```

Which is a part of `utilities.scss`